rclpy_examples/services/add_two_ints_client_async_py.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rclpy
+
+import sys
+
+from example_interfaces.srv import AddTwoInts
+
+
+def main(args=None):
+    if args is None:
+        args = sys.argv
```

Comment:
`rclpy.init(args)` already does this: https://github.com/ros2/rclpy/blob/master/rclpy/rclpy/__init__.py#L53